projects/DiffusionDet/diffusiondet/diffusiondet.py

```diff
@@ -0,0 +1,35 @@
+# Copyright (c) OpenMMLab. All rights reserved.
+
+# ========================================
+# Modified by Shoufa Chen
+# ========================================
+# Modified by Peize Sun, Rufeng Zhang
+# Contact: {sunpeize, cxrfzhang}@foxmail.com
+#
+# Copyright (c) Facebook, Inc. and its affiliates. All Rights Reserved
+
```

Comment:
This file seems to be created by us, thus, we only need to keep the copyright of open-mmlab.